.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,28 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Environmental Info:**
```

Comment:
The section titles and information we're asking for don't line up here. People will put the info where the hidden text is when filling out the issue, so uname -a should be after environment, k3s -v should be after k3s version, '3 masters' should be after cluster config.

Reply:
Fixed in https://github.com/rancher/k3s/pull/1767/commits/caf16a14832e490a0a14b7620b820a42b0fcd0b0